internal/config/constants.go

```diff
@@ -83,3 +83,12 @@ const (
 	EnvRegion     = "MINIO_REGION"      // legacy
 	EnvRegionName = "MINIO_REGION_NAME" // legacy
 )
+
+/*
+Expiration Token durations
+These values are used to validate the expiration time range from either the exp claim or MINI_STS_DURATION value
+*/
```

Comment:
```suggestion
// Expiration Token durations
// These values are used to validate the expiration time range from 
// either the exp claim or MINI_STS_DURATION value
```

In Go `//` is preferred over `/*` unlike C/C++